pom.xml

```diff
@@ -2240,7 +2240,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-shade-plugin</artifactId>
-                    <version>3.1.1</version>
+                    <version>3.2.1</version>
```

Comment:
Is there a reason this change has to be in this PR?

Reply:
I picked up this from the info a bytedance engineer provided.
Let me try to drop this.

Reply:
3.1.1 does not work with jdk11.

[2024-05-28T23:18:01.818Z] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:3.1.1:shade (default) on project presto-
spark-launcher: Error creating shaded jar: Problem shading JAR /home/jenkins/agent/workspace/prestodb_presto_PR-22656/presto-spark-

launcher/target/presto-spark-launcher-0.288.jar entry com/facebook/presto/spark/launcher/LauncherUtils.class: java.lang.IllegalArgumentException -> [Help 1]